runtime/strategies.md

```diff
@@ -0,0 +1,74 @@
+# Strategies
+
+The planner applies these strategies to produce resolved recipes to be instantiated in the Arc.
+
+##InitPopulation
+Loads recipes from arc’s context recipes.
+[init-population.js](https://github.com/PolymerLabs/arcs/blob/master/runtime/strategies/init-population.js)
+
+##ConvertConstraintsToConnections
+Converts connection constraints (eg ParticleA.viewA -> particleB.viewB) in the recipe into actual particles and view connections.
+Note: arrow direction are ignored at this time.
+[convert-constraints-to-connections.js](https://github.com/PolymerLabs/arcs/blob/master/runtime/strategies/convert-constraints-to-connections.js)
+
+##AssignViewsByTagAndType
+Maps recipe view with “use” fate to a local view in the arc matching type and tags.
+[assign-views-by-tag-and-type.js](https://github.com/PolymerLabs/arcs/blob/master/runtime/strategies/assign-views-by-tag-and-type.js)
+
+##AssignRemoteViews
+Maps recipe view with “map” fate to a remote view in the arc context matching type and tags.
+[assign-remote-views.js](https://github.com/PolymerLabs/arcs/blob/master/runtime/strategies/assign-remote-views.js)
+
+##CopyRemoteViews
+Maps recipe view with “copy” fate to a remote view in the arc context matching type and tags.
```

Comment:
I guess this is a mapping at this stage but maybe add something about the fact that a new view will be created on execution and the contents copied across, do differentiate this from AssignRemoteViews?

Reply:
done, thanks!